### DISCUSSION TITLE
Proxmox VE Kernel Clean TEST

### DISCUSSION THREAD

**Discussion by tteck** [tteck/Proxmox#1788] [Announcements]
I would greatly appreciate it if some experienced users could help verify the effectiveness of this script. While it has been successfully tested for removing a single old kernel, I believe it should work seamlessly for removing multiple old kernels as well. However, I would welcome your confirmation through testing.

```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/kernel-clean-test.sh)"
```
I'm pleased to share that the updated script offers the flexibility to select specific kernels for removal, unlike the current version, which is an all-or-nothing approach. As soon as I receive confirmation, I'll replace the current script with the updated version.

**Comment by Jefe2** [comment]
My system had two old kernels.  Running this script and selecting both entries successfully removed both kernels.

**Reply by tteck** [reply]
Thank you!
